backend/src/main/java/com/ambacam/model/Pays.java

```diff
@@ -0,0 +1,71 @@
+package com.ambacam.model;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@Entity
+public class Pays implements Serializable {
+
+	private static final long serialVersionUID = -8019398584857375323L;
```

Comment:
Please remove this serialVersionUID (it is the same as Role) and generate its own serialVersionUID (put your mouse on the name of the class and then choose "add generate serial version ID")